src/main/java/car_racing/InputView.java

```diff
@@ -0,0 +1,27 @@
+package car_racing;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final String INPUT_MESSAGE_NUMBER_OF_CARS = "자동차 대수는 몇 대 인가요 ?";
+    private static final String INPUT_MESSAGE_NUMBER_OF_ROUND = "시도할 회수는 몇 회 인가요 ?";
+
+    public int showMessageAndGetNumberOfCars() {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println(INPUT_MESSAGE_NUMBER_OF_CARS);
+        int numberOfCars = scanner.nextInt();
+
+        return numberOfCars;
+    }
```

Comment:
바로 return 해줘도 좋을 것 같아요! 😄 
```suggestion
    public int showMessageAndGetNumberOfCars() {
        Scanner scanner = new Scanner(System.in);

        System.out.println(INPUT_MESSAGE_NUMBER_OF_CARS);

        return scanner.nextInt();
    }
```